python/lsst/dm/OCPS/ocps_csc.py

```diff
@@ -92,13 +74,21 @@
 DONE_PHASES = ["completed", "error", "aborted", "unknown"]
 
 
+class OcpsIndex(enum.IntEnum):
+    LATISS = 1
+    LSSTComCam = 2
+    LSSTCam = 3
```

Comment:
I strongly suggest moving this to ts_idl. It is information available in ts_xml and we've found it useful to keep all of that in ts_idl so that users can access it without importing the CSC package.

Our usual convention is to name the enum `SalIndex` and put it in a module with the name of the SAL component (OCPS in this case). Then to use it in your code we suggest:
```
from lsst.ts.idl.enums.OCPS import SalIndex.
```
As opposed, say, to using `from lsst.ts.idl.enums import OCPS`, because OCPS looks like a symbol that should be defined in dm_OCPS. (I tried that a few times early on and quickly realized how confusing it was.)

Reply:
Aha!  It didn't seem great to have this here, given the duplication with ts_xml.

Would you mind if I started a "How to write an indexed CSC" doc, since I don't think there is one?

Reply:
I would prefer that you included the info in the "how to write a CSC" document in ts_salobj. The file is `salobj_cscs.rst` and the rendered text is here (as I'm sure you know): https://ts-salobj.lsst.io/salobj_cscs.html

It's starting to feel a bit unwieldy, what with all the options -- configurable or not, indexed or not, externally commandable or not (though externally commandable salobj-based CSCs are so rare that it hardly counts).

Reply:
I'll add something as part of https://github.com/lsst-ts/ts_salobj/pull/207